packages/twenty-front/src/modules/object-record/record-show/hooks/useRecordShowPagePagination.ts

```diff
@@ -130,22 +130,32 @@ export const useRecordShowPagePagination = (
     !isFirstRecord || (isFirstRecord && cacheIsAvailableForNavigation);
 
   const navigateToPreviousRecord = () => {
-    if (isFirstRecord) {
+    if (isFirstRecord || !recordBefore) {
```

Comment:
looks weird! 
isFirstRecord is already checking recordBefore

Reply:
Mmh I saw a bug when you clicked quickly, recordBefore wasn't loaded, I'll double check why this solved it

Reply:
OK you were right, this is a cleaner fix for the bug I saw: https://github.com/twentyhq/twenty/pull/9735/commits/39ba4ae56041ed02c01e049e91f1de07f2c13413